src/stackwalk.c

```diff
@@ -376,7 +376,8 @@ JL_DLLEXPORT jl_value_t *jl_lookup_code_address(void *ip, int skipC)
         jl_svecset(r, 5, jl_box_bool(frame.inlined));
         jl_svecset(r, 6, jl_box_long((intptr_t)ip));
     }
-    free(frames);
+    if (frames)
```

Comment:
I thought it was implementation-defined, but it looks like posix and MSVC all specify it to be a no-op.